src/login/lclif.p.h

```diff
@@ -56,7 +56,12 @@ enum login_packet_id {
 	//PACKET_ID_CA_SSO_LOGIN_REQa       = 0x825a, /* unused */
 
 	// AC (Login to Client)
+
+#if PACKETVER < 20170315
 	PACKET_ID_AC_ACCEPT_LOGIN         = 0x0069,
+#else
+	PACKET_ID_AC_ACCEPT_LOGIN         = 0x0ac4,
```

Comment:
But here other issue. If add another packet name, need add ifdefs inside lclif.c. For now it clean from ifdefs